.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+### How to use
+Leave the headings unless they don't apply to your PR; replace the explanatory text with text describing your PR.
+
+#### Summary
+A one-line description of your change that will be extracted and added to the [project changelog](../data/changelog.txt).  
+The format is: ```SUMMARY: Category "description"```  
+The categories to choose from are: Features, Content, Interface, Mods, Balance, Bugfixes, Performance, Infrastructure, Build, I18N
```

Comment:
Is there a description of these categories somewhere? I'm not sure of the difference between Features and Content or between Infrastructure and Build. I get that it's going to be a judgment call, but more explanation is necessary.

There should also be a category for Documentation only fixes, unless those are supposed to go under Interface or Infrastructure.

Two particular issues come to mind:
1. Code reorganization/refactoring. Build or Infrastructure?
2. Code that adds support to allow modders to do something, and gives an example. Is that Features, Content, Mods, or Infrastructure?

Reply:
Will there be correlation between Categories and current set of Labels?

Reply:
I think if we document each category, it's going to need to go in it's own document, that's going to be way too verbose for the template, just from the point of view of people having to delete all that content before entering their own.
I'll sketch something out at docs/ChangelogGuidelines.md

Reply:
The changelog guidelines look good to me. They're not too long but they're still comprehensive. Some stuff is still going to be a judgment call, but it's a judgment call that people have enough information to make.